contrib/rpm/bitcoin.spec

```diff
@@ -0,0 +1,346 @@
+%define bdbv 4.8.30
+
+Name:		bitcoin
+Version:	0.12.0
+Release:	0%{?dist}
+Summary:	Peer to Peer Cryptographic Currency
+
+Group:		Applications/System
+License:	MIT
+URL:		https://bitcoin.org/
+Source0:	https://bitcoin.org/bin/bitcoin-core-%{version}/bitcoin-%{version}.tar.gz
+Source1:	http://download.oracle.com/berkeley-db/db-%{bdbv}.NC.tar.gz
+
+Source10:	https://raw.githubusercontent.com/bitcoin/bitcoin/master/contrib/debian/examples/bitcoin.conf
+
+#man pages
+Source20:	https://raw.githubusercontent.com/bitcoin/bitcoin/master/contrib/debian/manpages/bitcoind.1
```

Comment:
I am unfamiliar with RPM spec files, but could this be made dynamic? e.g.

`https://raw.githubusercontent.com/bitcoin/bitcoin/v%{version}/contrib/debian/examples/bitcoin.conf`